cs251tk/student/markdownify/process_file.py

```diff
@@ -28,11 +28,20 @@ def get_file(filename, results, options):
     return True
 
 
-def compile_file(filename, steps, results, supporting_dir):
+def compile_file(filename, steps, results, supporting_dir, basedir, web, student):
+    server_path = ''
+
+    if web:
+        server_path += '-o "{}/server/server_file" '.format(basedir)
```

Comment:
Build this up as 

```python
server_path = ' '.join([
  "-o whatever".format,
  "".format(),
  …
])
```

instead of string concatenation, please?